### PR TITLE
GEL test bug fixes

### PR DESCRIPTION
**Pull request by sk-sahu**
Improvements - 
* Plot theme and colour changed to align with platform
* Refactor how data-frames being created dynamically
* Simplified `cb_list_cohort()` for user to get the complete list along side of default 10

Bug found during testing with GEL data -
* Use cohort browser API server as base-URL instead of cloudos base-URL (This is how the end points work in GEL at least for now)
* Now default columns directly come from back-end instead of hard coding them
* handling of `NULL` description in cohort in `cb_list_cohort()`
* value filter JSON fix

### CODE REVIEW

File: R/cb_extract_cohort.R [hunk 1]
```diff
@@ -82,16 +82,18 @@ cb_get_samples_table <- function(cloudos,
...
-                  body = list("pageNumber" = page_number,
-                              "pageSize" = page_size,
-                              "columns" = columns,
-                              "search" =  search,
-                              "returnTotal" = FALSE),
-                  encode = "json"
+                  body = jsonlite::toJSON(
+                              list("pageNumber" = page_number,
+                                  "pageSize" = page_size,
+                                  #"columns" = columns, # TODO
```
Comment (cgpu) [inline R/cb_extract_cohort.R:92]: Open an issue by referencing this to not forget the ToDo @sk-sahu

File: R/cb_json.R [hunk 2]
```diff
@@ -74,7 +76,7 @@
 
 ############################################################################################
 # Column JOSN
-# TODO work on column - not able to find end-point that returns this information
+# TODO work on column - At this point NO end-point that returns this information, there are cards
```
Comment (cgpu) [inline R/cb_json.R:79]: Add issue for this as well to keep track of this, and come back once the endpoint is implemented.

File: R/cb_plots.R [hunk 3]
```diff
@@ -66,10 +66,12 @@ cb_plot_filters <- function(cloudos, cohort){
...
-            theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1)) + 
+            theme_classic() +
+            theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1), 
+                  panel.grid.major.x = element_blank(),
+                  panel.grid.major.y = element_line( size=.1, color="black")) + 
             labs(title= fields_name) +
-            xlab(label = "range") + 
-            theme_bw()
+            xlab(label = "range")
         }else{
```
Comment (cgpu) [inline R/cb_plots.R:75]: We can refactor in next iteration of small fixes, to stop first, eg see example here:
https://speakerdeck.com/jennybc/code-smells-and-feels?slide=46

Add a low priority issue, eg P2.

File: man/cb_list_cohorts.Rd [hunk 4]
```diff
@@ -4,15 +4,13 @@
...
 \item{cloudos}{A cloudos object. (Required)
 See constructor function \code{\link{connect_cloudos}}}
 
-\item{page_number}{Number of page. (Optional) Default - 0}
-
-\item{page_size}{Number of entries in a page. (Optional) Default - 10}
+\item{size}{Number of cohort entries from database. (Optional) Default - 10}
 }
 \value{
 A data frame with available cohorts.
```
Comment (cgpu) [inline man/cb_list_cohorts.Rd:16]: Below the png files (sorry for misplacing, cannot comment if binary 😄 )
are they escaping from vignette building and need to be gitignored or we are good and keeping them for reference in the repo?

eg `man/figures/README-unnamed-chunk-10-1.png`

Either is fine, only checking if intended.

**Review by cgpu — APPROVED** [review]
Looks good to me 👍 

Only added 3 notes for the **ToDo** and a refactoring opportunity. Feel free to squash once you \
- [x] check comments
- [x] open issues for 3 bits mentioned above

File: R/cb_json.R (hunk 2 above)
Comment (sk-sahu) [reply R/cb_json.R:79]: Done https://github.com/lifebit-ai/cloudos/issues/10

File: R/cb_extract_cohort.R (hunk 1 above)
Comment (sk-sahu) [reply R/cb_extract_cohort.R:92]: Done https://github.com/lifebit-ai/cloudos/issues/10

File: R/cb_plots.R (hunk 3 above)
Comment (sk-sahu) [reply R/cb_plots.R:75]: Yup good point. I have opened an issue, not to forget - https://github.com/lifebit-ai/cloudos/issues/11

File: man/cb_list_cohorts.Rd (hunk 4 above)
Comment (sk-sahu) [reply man/cb_list_cohorts.Rd:16]: We can keep them. Majority of pkgdown rendered things goes to [gh-pages](https://github.com/lifebit-ai/cloudos/tree/gh-pages) branch. These are used as cache. 

ATM these are not big files, But on later point we can remove them.

**Comment by sk-sahu** [comment]
Thank you @cgpu :raised_hands: :)

**Pull request merged**